entity-types/infra-elasticsearchnode/definition.yml

```diff
@@ -1,5 +1,46 @@
 domain: INFRA
 type: ELASTICSEARCHNODE
+
+synthesis:
+  # Apart from this rule ELASTICSEARCH entities are also created from infra pipeline.
+  rules:
+  - compositeIdentifier:
+      separator: ":"
+      attributes:
+        - elasticsearch.cluster.name
+        - elasticsearch.node.name
+    name: elasticsearch.node.name
+    encodeIdentifierInGUID: true
+    conditions:
+      - attribute: eventType
+        value: Metric
+      # Some metrics from the receiver start with the 'elasticsearch.' prefix.
+      # The receiver also collects metrics that start with 'jvm.' prefix.
+      - attribute: metricName
```

Comment:
this is not an OR. all conditions must be satisfied so this will never match.

Our recomendation for now is to have 2 rules, one checking the elasticsearch. and another checking the jvm.

Reply:
I would also suggest to add some tests since those would have catch this issues and will prevent new rules to override yours by mistake.

Example
https://github.com/newrelic/entity-definitions/tree/main/entity-types/ext-pihole/tests

Reply:
Could you check this comments @reese-lee ?

Reply:
Hi @reese-lee, any updates on this?

Reply:
Hello @marcsanmi, no updates quite yet, I will need some time to add the tests. I apologize about the long delay with this, but I should have some time at the beginning of Q1 to return to this PR.